third_party/docfx/templates/devsite/partials/class.tmpl.partial

```diff
@@ -18,7 +18,7 @@
 {{#overload}}
 <a id="{{id}}" data-uid="{{uid}}"></a>
 {{/overload}}
-<h3 id="{{id}}" data-uid="{{uid}}">{{name.0.value}}</h3>
+<h3 id="{{id}}" data-uid="{{uid}}" translate="no">{{name.0.value}}</h3>

```

Comment:
Let's do `class="notranslate"` instead. I think the `translate` attribute will get stripped out.

Reply:
done!